static/app/views/performance/trends/utils.tsx

```diff
@@ -270,6 +276,10 @@ export function modifyTrendView(
     if (query.hasFilter('transaction.duration')) {
       query.removeFilter('transaction.duration');
     }
+
+    if (trendParameter.column && query.hasFilter(trendParameter.column)) {
+      query.removeFilter(trendParameter.column);
+    }
```

Comment:
apparently, we're applying filters like `measurements.lcp:>15min` and that's not supported by the metrics dataset